documentation/docs/development-plan/schedule.md

```diff
@@ -128,108 +128,64 @@ gantt
 ## Milestone Demo Features
 
 ### Milestone Demo 1 [Account Creation & Pet Selection]
-F: User can create/log in to account
 
-R: User can enter credentials
+- F1: Account Management
+    - R1: User is able to register new account.
+    - R2: User is able to login with an existing account.
+    - R3: User is able to update their account name, and email address.
+    - R4: User is able to view their account information.
+    - R5: Credentials are authenticated via JWT token.
+    - R6: First-time user will be sent a verification email.
 
-R: Credentials are authenticated via JWT token
+- F2: Pet Selection, Customization, and Care
+    - R1: User can choose between two virtual pets.
+    - R2: User can select between two different cats (color palette).
+    - R3: User can name their virtual pet.
+    - R4: Users can see a still image of their virtual pet at all times.
```

Comment:
R5: User sees the pet's mood change temporarily from neutral to happy when they click on it.